fetch.bs

```diff
@@ -2127,12 +2133,24 @@ identified by a <b>key</b> (a <a>network partition key</a>), an <b>origin</b> (a
 <a for=/>origin</a>), and <b>credentials</b> (a boolean).
 
 <p>To <dfn export id=concept-connection-obtain>obtain a connection</dfn>, given a <var>key</var>,
-<var>origin</var>, and <var>credentials</var>, run these steps:
+<var>origin</var>, <var>credentials</var>, optionally <var>forHTTP3</var> (a boolean, false by
+default), and optionally <var>sharable</var> (a boolean, true by default), run these steps:
```

Comment:
Beware that this will conflict with @yoichio's work in #1141. With 3 optional parameters, we probably need to make them non-positional. See https://infra.spec.whatwg.org/#algorithm-params for syntax (the fetch algorithm has them as well now). Note that parameters that default to true are not allowed.

Reply:
I see.  I'm O.K. as first-come-first-out.

Reply:
Done.